aws-sagemaker-python/Pulumi.yaml

```diff
@@ -0,0 +1,11 @@
+name: ${PROJECT}
+description: ${DESCRIPTION}
+runtime: python
+template:
+  description: Deploy a Hugging Face LLM model to AWS SageMaker with CloudWatch monitoring
```

Comment:
In other architecture templates we've also included the language in the description so that Metabase queries can be used to help roughly determine language usage

Reply:
Also, it should be "Amazon SageMaker"

Reply:
resolved with https://github.com/pulumi/templates/pull/643/commits/85e3b6c29efe928c45b0b474f23fed1b6fc12ee6